lib/utils/Integers.ml

```diff
@@ -0,0 +1,207 @@
+let _ =
+  if Sys.int_size < 33 then
+    failwith "The size of the integers available is not large enough."
+
+(** First possible address : 0 *)
+let first_address = 0
+
+(** First possible address : 2^32 - 1 *)
+let last_address = (1 lsl 32) - 1
+
+let is_uint16 =
+  let two_16 = 1 lsl 16 in
+  fun i -> 0 <= i && i < two_16
+
+let is_int24 =
```

Comment:
Même commentaire

Reply:
Fixed in ecf8949a2acd1b3f3c7509bc4fcc6ff07f12e557.